.github/workflows/ci.yml

```diff
@@ -21,8 +21,9 @@ jobs:
           python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
+          sudo add-apt-repository ppa:canonical-server/server-backports
```

Comment:
FWIW, I can still reproduce the crash on the latest QEMU (6.2.0), so I think we can drop the changes to update QEMU.